manual/cops_lint.md

```diff
@@ -657,6 +657,8 @@ end
 
 ## Lint/ErbNewArguments
 
+Required Ruby version: 2.6
```

Comment:
We have admonitions enabled and this seems like a good use for them. You can some `!!! Note` prefix to this to turn it into an admonition.

Reply:
Great suggestion! This prefix feature is what I wanted thing :-)
I updated this one. Thank you always!